.travis.yml

```diff
@@ -8,7 +8,7 @@ jdk: openjdk8
 
 env:
   global:
-    - DESTINATION_IMAGE_NAME="$DOCKER_GCP_LOCATION/census-rm-actionexportersvc"
+    - DESTINATION_IMAGE_NAME="eu.gcr.io/census-rm-ci/rm/census-rm-actionexportersvc"
```

Comment:
Should we remove this `env.global` section now as `DESTINATION_IMAGE_NAME` is only being used once? I guess it does it a meaningful name, though.